src/playground/package.json

```diff
@@ -20,6 +20,6 @@
     "@codemirror/view": "^6.23.0",
     "@lezer/highlight": "^1.2.0",
     "@lezer/javascript": "^1.4.13",
-    "ezno": "=0.0.21"
+    "ezno": "^0.0.22"
```

Comment:
it'd be great if the playground mentioned the version it's using 👀

Reply:
Check line 162 :)